samples/subsys/tracing/sample.yaml

```diff
@@ -8,7 +8,7 @@ common:
         - "thread_b: Hello World from (.*)!"
 tests:
   tracing.format.sysview:
-    platform_allow: nrf52840dk_nrf52840
+    filter: CONFIG_HAS_SEGGER_RTT
```

Comment:
This filter isn't working, but I don't understand why. @nashif will you please have a look?